waltham: Add documentation for waltham

tnaoko opened the pull request:
Please kindly check it.

Signed-off-by: Naoko Tanibata <tnaoko@de.adit-jv.com>

tnaoko committed:
waltham: Add documentation for waltham

Signed-off-by: Naoko Tanibata <tnaoko@de.adit-jv.com>

benierc commented:
tnaoko,

in addition, to the fact that i think it is not the right place for this documentation, 
thanks to @sebd71 there are some issues with your markdown files,

you should remove headers from markdowns, it will generate ugly results in documentation website, for example remove this:

\---
title: AGL - Waltham Overview
author: Naoko Tanibata (Advanced Driver Information Technology)
date: 2019-12-18

categories: architecture
tags: architecture, graphics, wayland, weston, waltham
layout: techdoc

\---


there are also typo mistakes in your markdow, 

You can test your documentation locally, see the README here https://github.com/automotive-grade-linux/docs-webtemplate.

Don't hesitate if you have questions

kooltux commented:
The header in markdown should work but there may be an issue with spaces in it. Internally in doc tools, such headers are used or generated so they should be kept as much as possible.

tnaoko commented:
Hello, 
Thank you for checking this PR, but this PR should not be merged yet since I'm still working on fixing reviewed points. The below question is open, I've sent it to AGL mailing list.

> Thank you for the review. Based on your comments, I removed headers and moved Waltham documents src to weston-ivi-plugins repo on gerrit.
> Now I am trying to setup local AGL website to check my docs’ outlook but I am still struggling with it.
> I have Waltham book file and documentation files on weston-ivi-plugins(dev branch, gerrit), I want to fetch them from section_master.yml on docs-webtemplate (dev branch, github).
> I could build and run AGL website locally, but it does not have linjk to waltham documents.
> I think I have misunderstanding on  documentation-from-local-repositories . Would you suggest how I can achieve it?
> 
> 1. Waltham book file  and document src files  in my dev branch on weston-ivi-pluigins(gerrit)
> https://gerrit.automotivelinux.org/gerrit/gitweb?p=src%2Fweston-ivi-plugins.git;a=shortlog;h=refs%2Fheads%2Fsandbox%2Ftnaoko%2Fdevel_waltham-transmitter
> 
> 2. Modified section_master.yml in my dev branch on github, which try to fetch #1.
> https://github.com/tnaoko/docs-webtemplate
